sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/SupportsNamespaces.java

```diff
@@ -53,6 +53,16 @@
    */
   String PROP_COMMENT = "comment";
 
+  /**
+   * A property to specify the owner of the namespace.
+   */
+  String PROP_OWNER_NAME = "ownerName";
+
+  /**
+   * A property to specify the type of the namespace's owner.
+   */
+  String PROP_OWNER_TYPE = "ownerType";
+
   /**
    * The list of reserved namespace properties.
```

Comment:
shall we update the reverted properties too?

Reply:
How about after supporting `SET OWNER`